tests/basic/pure_test.go

```diff
@@ -4143,3 +4143,129 @@ var _ = Describe("{CreateNewPoolsWhenFadaFbdaVolumeCreationInProgress}", func()
+				StorageProvisioner: fmt.Sprintf("%v", portworx.PortworxCsi),
+				PvcSize:            6 * units.GiB,
+			})
+			log.FailOnError(err, "Failed to schedule application of %v namespace", "test-fada-volume")
+			contexts = append(contexts, context...)
+
+		})
+		ValidateApplications(contexts)
+
+		stepLog = "Do parallel resize of base apps volume and FADA volume"
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+
+			go func() {
+				for _, ctx := range contexts[:1] {
+					var appVolumes []*volume.Volume
+					var err error
+					var pvcs []*v1.PersistentVolumeClaim
+					appVolumes, err = Inst().S.GetVolumes(ctx)
+					log.FailOnError(err, "Failed to get volumes from context")
+					log.InfoD(fmt.Sprintf("increase volume size %s on app %s's volumes: %v",
+						Inst().V.String(), ctx.App.Key, appVolumes))
+					for _, eachVol := range appVolumes {
+						pvc, err := GetPVCObjFromVol(eachVol)
+						log.FailOnError(err, "Failed to get PVC Details from Volume [%v]", eachVol.Name)
+						pvcs = append(pvcs, pvc)
+
+					}
+					for _, pvc := range pvcs {
+						pvcSize := pvc.Spec.Resources.Requests.Storage().String()
+						pvcSize = strings.TrimSuffix(pvcSize, "Gi")
+						pvcSizeInt, err := strconv.Atoi(pvcSize)
+						log.InfoD("increasing pvc [%s/%s]  size to %v %v", pvc.Namespace, pvc.Name, 2*pvcSizeInt, pvc.UID)
+						resizedVol, err := Inst().S.ResizePVC(ctx, pvc, uint64(pvcSizeInt))
+						log.FailOnError(err, "pvc resize failed pvc:%v", pvc.UID)
+						log.InfoD("Vol uid %v", resizedVol.ID)
+						resizedVols = append(resizedVols, resizedVol)
+					}
+
+				}
+
+			}()
+			for _, ctx := range contexts[1:] {
+				var appVolumes []*volume.Volume
+				var err error
+				var pvcs []*v1.PersistentVolumeClaim
+				appVolumes, err = Inst().S.GetVolumes(ctx)
+				log.FailOnError(err, "Failed to get volumes from context")
+				log.InfoD(fmt.Sprintf("increase volume size %s on app %s's volumes: %v",
+					Inst().V.String(), ctx.App.Key, appVolumes))
+				for _, eachVol := range appVolumes {
+					pvc, err := GetPVCObjFromVol(eachVol)
+					log.FailOnError(err, "Failed to get PVC Details from Volume [%v]", eachVol.Name)
+					pvcs = append(pvcs, pvc)
+
+				}
+				for _, pvc := range pvcs {
+					pvcSize := pvc.Spec.Resources.Requests.Storage().String()
+					pvcSizestring := strings.TrimSuffix(pvcSize, "Gi")
+					pvcSizeInt, err := strconv.Atoi(pvcSizestring)
+					log.InfoD("increasing pvc [%s/%s]  size to %v %v", pvc.Namespace, pvc.Name, 2*pvcSizeInt, pvc.UID)
+					resizedVol, err := Inst().S.ResizePVC(ctx, pvc, uint64(pvcSizeInt))
+					log.FailOnError(err, "pvc resize failed pvc:%v", pvc.UID)
+					log.InfoD("Vol uid %v", resizedVol.ID)
+					resizedVols = append(resizedVols, resizedVol)
+				}
+			}
+		})
+
+		stepLog = "Validate volume resize on both base volume and FADA volume"
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+			var err error
+			log.FailOnError(err, "Failed to get volumes from context")
```

Comment:
why are we checking err even if we are not returning anything ?

Reply:
My bad removed it